File: programmers/shim/Java/level1/20220603_2016년.java
package Study_2022.programmers.shim.Java.level1;

class Solution {
    public String solution(int a, int b) {
       String W[]={"THU","FRI","SAT","SUN","MON","TUE","WED"};
       int M[]={31,29,31,30,31,30,31,31,30,31,30,31};
        for(int i=0; i<a-1; i++)
            b+=M[i];
        return W[b%7];
    }
}

File: programmers/shim/Python/level1/20220603_2016년.py
def solution(a, b):
    W=['THU','FRI','SAT','SUN','MON','TUE','WED']
    M=[31,29,31,30,31,30,31,31,30,31,30,31]
    return W[(sum(M[i]for i in range(a-1))+b)%7]